### ISSUE TITLE
Enable hardware checks automatically based on chassis model

### ISSUE THREAD

**Issue by BjoernT** [rcbops/rpc-openstack#1729]
##### What type of issue is this? Please tick the appropriate box.
- [ ] bug - existing functionality that does not work for you as documented
- [x] feature - new functionality that you would like to see added

##### What is the expected behavior?

Currently the hardware monitors need to be enabled manually and therefore are easily forgotten.
Based on ansible facts we can enable those hardware checks automatically, matching certain chassis vendor.

##### How can we reproduce this issue?

N/A

##### How can we verify your issue is addressed?

N/A

##### If you are using or know of a workaround please describe the necessary steps?

N/A

##### What is your evaluation on the importance of this issue, and why?

Prio 3
----------------------------------------------------------------------------------------------------------------------------
###### For each pull request, update the branch line to reference it.
- [ ] master
- [ ] newton-14.0
- [ ] mitaka-13.1

##### Upgrade impact. Consider the impact on moving between major, minor and patch releases.
- [ ] This change does not impact upgrades
- [ ] This change addresses a potential upgrade impact
- [ ] The upgrade impact is being tracked by another issue.

The issue tracking the upgrade impact is:

##### Testing impact
- [ ] An issue/pull-request has been create or exists to track the testing gap, and is referenced below
- [ ] This change includes additional testing
- [ ] This change is tested by the existing setup

Additional testing links:

##### Release note
- [ ] This change includes a release note.
- [ ] This change does not require a release note.

##### Does your change require documentation?
- [ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
- [ ] Yes, and I contributed a documentation patch to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
- [ ] No documentation required for this change

Additional documentation links (please add links if you have ticked yes):

**Comment by BjoernT** [comment]
This is getting fixed with https://github.com/rcbops/rpc-openstack/pull/1727

**Comment by alextricity25** [comment]
Thanks @BjoernT for looking into this. Could you do us a favor and update this issue card once the fix is in place? Thanks!

**Comment by BjoernT** [comment]
sure

**Comment by antonym** [comment]
#1727 has landed.

**Comment by antonym** [comment]
This has merged and been backported to newton, setting host_check: true will use Ansible to determine what the host is and apply the proper checks.
